### DISCUSSION TITLE
Export to JQ

### DISCUSSION THREAD

**Discussion by danielsinai** [react-querybuilder/react-querybuilder#532] [Ideas]
Hi 👋 
I like the library a lot! 
I have a use-case of exporting the query to JQ format. Is it possible?

**Comment by jakeboone02** [comment]
Hi @danielsinai, I'm glad you like the library!

Are you referring to the [`jq` command line JSON processor](https://jqlang.github.io/jq/)? If so, could you provide an example of a query builder query (screenshot or query object is fine) along with the expected output of `formatQuery` in `jq` format?

I haven't used `jq` but after looking quickly at the syntax, I have the same concerns about it as I do with GraphQL, namely that the "what" can be intermingled with the "which". RQB really only deals with the "which" (like the `WHERE` clause in SQL), not the "what" (like the `SELECT` clause in SQL).

But if you can provide a good "from this RQB query I'd like to see this `jq` query" example I may be able to help.

**Reply by danielsinai** [reply]
First of all thanks for the quick response!

I'll research and share here an example 🚀
